package.json

```
@@ -15,6 +15,7 @@
   "scripts": {
     "build": "turbo run build",
     "build:ui": "turbo run build --filter=./packages/ui",
+    "build:storybook": "turbo run build:storybook --filter=./packages/ui",
     "build:website": "turbo run build --filter=./apps/website",
```

```suggestion
    "build:ui": "turbo run build --filter=swapr-ui",
    "build:storybook": "turbo run build:storybook --filter=swapr-ui",
    "build:website": "turbo run build --filter=swapr-ui-website",
```

This looks cleaner to me, what do you think?

---

Yes, updated.